app/services/TradingPremisesService.scala

```diff
@@ -61,15 +82,35 @@ class TradingPremisesService {
       }
     }
 
-
-  def patchTradingPremisesBusinessActivities(tradingPremises: Seq[TradingPremises])
-                                            (fn: ((WhatDoesYourBusinessDo, Int) => WhatDoesYourBusinessDo)): Seq[TradingPremises] =
+  private def patchTradingPremisesBusinessActivities(tradingPremises: Seq[TradingPremises])
+                                                    (fn: ((WhatDoesYourBusinessDo, Int) => WhatDoesYourBusinessDo)): Seq[TradingPremises] = {
     tradingPremises.zipWithIndex map { case (tp, index) =>
-      tp.whatDoesYourBusinessDoAtThisAddress(
-        tp.whatDoesYourBusinessDoAtThisAddress.fold(WhatDoesYourBusinessDo(Set.empty)){ wdybd =>
+      val stuff: TradingPremises = tp.whatDoesYourBusinessDoAtThisAddress {
```

Comment:
Can we rename this to something other than `stuff`?

Reply:
Done